docker-compose.yml

```diff
@@ -46,6 +46,7 @@ services:
       args:
         - DOCKER_REPOSITORY=${dockerHubUsername:-dfedigital}
```

Comment:
don't think you need that now...
or any of the args looks defaulted in dockerfile

Reply:
Yeah... I don't think so either but we decided to leave it for now and just focus on what was broken.